clients/cobol-lsp-vscode-extension/src/test/runTest.ts

```diff
@@ -52,6 +52,11 @@ async function main() {
     }
     // run tests
     await runTests(options);
+    if (process.platform === "win32") {
+      console.log("----- RUNNING TEST USING UNC PATH ------");
+      options.extensionTestsEnv = { UNC_PREFIX: "\\." };
+      await runTests(options);
```

Comment:
Is it ok to run the UN C tests like this? or handle it using custom mocha test runner or something in that line

Reply:
In general, I think this would be fine. But the UNC path needs to be passed to vscode e.g.
```
launchArgs[0] = "\\\\.\\" + launchArgs[0];
```

Reply:
Thanks! let me try that

Reply:
Found below issue while adding UNC tests -
- Java doesn't mind path like `\\<hostname>\cobolls-test` , but **FAILS** for path like `\\.\c:\cobolls-test`

`Caused by: java.nio.file.InvalidPathException: Illegal character [:] in path at index 5: \\.\c:\cobolls-test
`
Also, there are a few bugs around UNC path [JDK-8218618](https://bugs.openjdk.org/browse/JDK-8218618), which results in a failure at random places. For e.g. this bug resulted me in an error for `UUID.randomUUID(`), as it internally relies on some nio packages.

Reply:
Is it possible that you are running unpatched version of java? The issue, you linked, mentions two v11 versions that are supposed to be fixed.

Reply:
I was even able to read an NTFS alternate stream using UNC path e.g. `\\.\c:\Source\test.txt:XYZ`